### PR TITLE
Update interactive instructions

### LINKED ISSUE #13345: You assume too much
This webpage clearly says that these are "interactive lessons that you can run in your browser". There is no mention of Visual Studio. When you start the hello world tutorial, nowhere does it tell you to open Visual Studio. 

So pretend you are on step 1 of the hello world tutorial. The very first instruction at the top of the page makes it sound like you need to type something into an interactive window of THE BROWSER. 

"Run the following code in the interactive window. To do that, type the following code block in the interactive window and click the Run button:"

You need to alter the instructions to include a mention of Visual Studio.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a62fb2fc-295c-d837-7d89-0835702a75d9
* Version Independent ID: f57c1676-b294-bce7-6b71-e4939371e8bb
* Content: [C# Tutorials](https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/#feedback)
* Content Source: [docs/csharp/tutorials/index.md](https://github.com/dotnet/docs/blob/master/docs/csharp/tutorials/index.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### LINKED ISSUE #14078: Expand explanation on how to use the interactive code editor
There lacks a clear set of instructions on how to open and use the code editor. A user might find their way into the editor if they have interacted with similar features before, but [as stated on the Intro to C# index page:](https://github.com/dotnet/docs/blob/master/docs/csharp/tutorials/intro-to-csharp/index.md#L15)

>  These first lessons assume no prior knowledge of programming or the C# language.

The first mention of using the code editor is here: https://github.com/dotnet/docs/blob/master/docs/csharp/tutorials/intro-to-csharp/hello-world.yml#L21

> Run the following code in the interactive window. To do that, type the following code block in the interactive window and click the **Run** button:

The interactive window is not visible at this point, and won't be until one presses a button labeled "Enter Focus Mode", which is not mentioned anywhere in this tutorial.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 3d841900-8699-af9b-f879-c5cf9b54f21e
* Version Independent ID: 2d85eae9-01e1-17af-b282-715ec3dc0dcb
* Content: [Hello World - Introduction to C# interactive C# tutorial](https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/intro-to-csharp/hello-world?tutorial-step=1)
* Content Source: [docs/csharp/tutorials/intro-to-csharp/hello-world.yml](https://github.com/dotnet/docs/blob/live/docs/csharp/tutorials/intro-to-csharp/hello-world.yml)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by BillWagner**
The instructions for some of our interactive tutorials don't match the latest site updates.
Update the first example instructions to tell users how to enter focus mode, and begin using the interactive window.

Fixes #13345
Fixes #14078

### CODE REVIEW

**Review by Thraka — APPROVED** [review]
LGTM You could change the text of the others to match the first regarding the button: "Select" being the key word instead of "click"

File: docs/csharp/tutorials/exploration/interpolated-strings.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ items:
 - title: Create an interpolated string
   durationInMinutes: 2
   content: |
-    Run the following code in the interactive window. To do that, type the code in the interactive window (replace `<name>` with your name) and select **Run**:
+    Run the following code in the interactive window. Click the **Enter Focus Mode** button. Then, type the following code block in the interactive window (replace `<name>` with your name) and select **Run**:
```
Comment (pkulikov) [inline docs/csharp/tutorials/exploration/interpolated-strings.yml]:
```suggestion
    Run the following code in the interactive window. Select the **Enter focus mode** button. Then, type the following code block in the interactive window (replace `<name>` with your name) and select **Run**:
```

File: docs/csharp/tutorials/intro-to-csharp/branches-and-loops.yml [hunk 2]
```diff
@@ -18,7 +18,7 @@ items:
 - title: Make decisions using the if statement
   durationInMinutes: 4
   content: |
-    Run the following code in the interactive window. To do that, type the following code block in the interactive window and click the **Run** button:
+    Run the following code in the interactive window. Click the **Enter Focus Mode** button. Then, type the following code block in the interactive window and click the **Run** button:
```
Comment (pkulikov) [inline docs/csharp/tutorials/intro-to-csharp/branches-and-loops.yml]:
```suggestion
    Run the following code in the interactive window. Select the **Enter focus mode** button. Then, type the following code block in the interactive window and select **Run**:
```

File: docs/csharp/tutorials/intro-to-csharp/hello-world.yml [hunk 3]
```diff
@@ -18,7 +18,7 @@ items:
 - title: Run your first C# program
   durationInMinutes: 2
   content: |
-    Run the following code in the interactive window. To do that, type the following code block in the interactive window and click the **Run** button:
+    Run the following code in the interactive window. Click the **Enter Focus Mode** button. Then, type the following code block in the interactive window and click the **Run** button:
```
Comment (pkulikov) [inline docs/csharp/tutorials/intro-to-csharp/hello-world.yml]:
```suggestion
    Run the following code in the interactive window. Select the **Enter focus mode** button. Then, type the following code block in the interactive window and select **Run**:
```

File: docs/csharp/tutorials/intro-to-csharp/list-collection.yml [hunk 4]
```diff
@@ -16,7 +16,7 @@ items:
 - title: Create lists
   durationInMinutes: 2
   content: |
-    Run the following code in the interactive window. To do that, type the following code block in the interactive window (replace `<name>` with your name) and click the **Run** button:
+    Run the following code in the interactive window. Click the **Enter Focus Mode** button. Then, type the following code block in the interactive window (replace `<name>` with your name) and click the **Run** button:
```
Comment (pkulikov) [inline docs/csharp/tutorials/intro-to-csharp/list-collection.yml]:
```suggestion
    Run the following code in the interactive window. Select the **Enter focus mode** button. Then, type the following code block in the interactive window (replace `<name>` with your name) and select **Run**:
```

File: docs/csharp/tutorials/intro-to-csharp/numbers-in-csharp.yml [hunk 5]
```diff
@@ -18,7 +18,7 @@ items:
 - title: Explore integer math
   durationInMinutes: 4
   content: |
-    Run the following code in the interactive window. To do that, type the following code block in the interactive window and click the **Run** button:
+    Run the following code in the interactive window. Click the **Enter Focus Mode** button. Then, type the following code block in the interactive window and click the **Run** button:
```
Comment (pkulikov) [inline docs/csharp/tutorials/intro-to-csharp/numbers-in-csharp.yml]:
```suggestion
    Run the following code in the interactive window. Select the **Enter focus mode** button. Then, type the following code block in the interactive window and select **Run**:
```

**Review by pkulikov — COMMENTED** [review]
"Click" -> "Select" is according to the docs guidelines:
https://docs.microsoft.com/en-us/style-guide/procedures-instructions/describing-interactions-with-ui

The name of the button is sentence-cased:
https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/intro-to-csharp/list-collection?tutorial-step=1

**Pull request merged**